scripts/gff.py

```diff
@@ -321,6 +348,9 @@ def filter_to_strand(input_path: str, output_path: str, strand: str) -> None:
 
     # Read GFF file
     features = load_gff(input_path)
+    # This line was added in the new file, so we keep it.
+    if "passplantcadfilter" in features.columns:
```

Comment:
This is not a good place to handle that corner case and casting to string is dangerous.  Why did you need this?  I think we should remove it unless there's a good reason for it.  The comment is confusing too -- I don't see how it's related to the code.